apel_rest/urls.py

```diff
@@ -11,6 +11,6 @@
     # url(r'^blog/', include('blog.urls')),
 
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^accounting-server/api/v1/cloud/record$', CloudRecordView.as_view()),
-    url(r'^accounting-server/api/v1/cloud/record/summary$', CloudRecordSummaryView.as_view())
+    url(r'^api/v1/cloud/record$', CloudRecordView.as_view()),
```

Comment:
Yep.... it seems to be a strange artifact of the kubernetes. The Apache-Django server handling the requests is being passed /api/..... urls (even though the url is /accounting-server/api/....).

However, when the Apache-Django looks for the static files (the files that make the Django Browseable interface) it only finds them if it thinks they're under /accounting-server/static rather than just /static/